miden/src/cli/run.rs

```diff
@@ -38,8 +38,12 @@ pub struct RunCmd {
     output_file: Option<PathBuf>,
 
     /// Enable tracing to monitor execution of the VM
-    #[clap(short = 't', long = "tracing")]
-    tracing: bool,
+    #[clap(short = 't', long = "trace")]
```

Comment:
@bobbinth Should we do the same in `prove.rs`?

Reply:
yes - good idea!

Reply:
@bobbinth done, I think I addressed all the comments that I have seen